array.js

```diff
@@ -0,0 +1,601 @@
+};
+
+
+// 2. С помощью двух вложенных циклов нарисуйте следующую пирамидку:
+
+for (let i = 1; i < 10; i++) {
+    document.write(i.toString().repeat(i) + "<br>");
+};
+
+// 3. Нарисуйте пирамиду, как показано на рисунке, воспользовавшись циклом
+
+let a = " ";
+for (let i = 1; i <= 12; i++) {
+    a += "**"
+    document.write(a.toString() + "<br>");
+};
+
+
+// 4. Пирамидка с цифрами
+
+for (let i = 1; i < 10; i++) {
+    document.write(i.toString().repeat(3) + "<br>");
+};
+
+// 5. Напишите функцию goThroughArray(list), которая проходится по массиву и выводит каждый элемент.
+
+let list = "1,2,3"
+
+function goThroughArray(list) {
+    return list.split(",")
+}
+
+console.log(goThroughArray(list))
+
+// 6. Даны два массива: ['a', 'b', 'c'] и[1, 2, 3].Объедините их вместе.
+
+let arr = ['a', 'b', 'c'];
+
+let array = [1, 2, 3];
+
+console.log(arr.concat(array));
+
+
+// 7. Дан массив['a', 'b', 'c'].Добавьте ему в конец элементы 1, 2, 3.
+
+let arr = ['a', 'b', 'c'];
+
+console.log(arr.concat([1, 2, 3]));
+
+// 8. Дан массив[1, 2, 3, 4, 5].С помощью метода splice преобразуйте массив в[1, 4, 5].
+
+let arr = [1, 2, 3, 4, 5];
+
+arr.splice(1, 2);
+console.log(arr);
+
+// 9. Дан массив[1, 2, 3, 4, 5].С помощью метода splice запишите в новый массив элементы[2, 3, 4].
+
+let arr = [1, 2, 3, 4, 5];
+
+arr.splice(0, 1);
+arr.splice(3);
+console.log(arr);
+
+// 10. Дан массив[1, 2, 3, 4, 5].С помощью метода splice сделайте из него массив[1, 2, 3, 'a', 'b', 'c', 4, 5].
+
+let arr = [1, 2, 3, 4, 5];
+arr.splice(2, 2, 'a', 'b', 'c');
+console.log(arr);
+
+// 11. Дан массив[1, 2, 3, 4, 5].С помощью метода splice сделайте из него массив[1, 'a', 'b', 2, 3, 4, 'c', 5, 'e'].
+
+let arr = [1, 2, 3, 4, 5];
+
+arr.splice(1, 0, 'a', 'b');
+console.log(arr);
+
+arr.splice(6, 0, 'c');
+console.log(arr);
+
+arr.splice(8, 0, 'e');
+console.log(arr);
+
+// 12. Дан массив[3, 4, 1, 2, 7].Отсортируйте его.let arr = [3, 4, 1, 2, 7];
+
+arr.sort((a, b) => a - b);
+
+console.log(arr);
+
+// 13. Дан объект { js: 'test', jq: 'hello', css: 'world' }.Получите массив его ключей.
+
+let obj = {
+    js: 'test',
+    jq: 'hello',
+    css: 'world',
+};
+
+console.log(Object.keys(obj));
+
+// 14. Дана строка, например, '123456'.Переверните эту строку(сделайте из нее '654321') не используя цикл.
+
+let str = "123456";
+
+result = str.split("").reverse().join("");
+
+console.log(result);
+
+// 15. Дана строка.Сделайте заглавным первый символ этой строки не используя цикл.Найдите два решения
+let str = "hello";
+
+result = str[0].toUpperCase() + str.slice(1);
+
+console.log(result);
+
+// 16. Проверьте, что строка начинается на http: //.
+
+let str = "https://docs.google.com/document/d/1EMA9Nm/edit";
+
+console.log(str.split("//"));
+
+let getResult = (str) => (str[0] === "http:");
+
+console.log(getResult(str));
+
+// 17. Проверьте, что строка заканчивается на.html
+
+let str = " https://github.com/Kartynnikk/js.html";
+
+let getResult = (str) => str.includes(".html");
```

Comment:
```suggestion
let getResult = (str) => str.includes(".html");
```

try this

" https://github.com/Kartynnikk.html/js.pdf";

Reply:
it will work, even though it shouldn't

Reply:
edited